src/display/text_layer.js

```diff
@@ -57,11 +57,66 @@ const renderTextLayer = (function renderTextLayerClosure() {
 
   const NonWhitespaceRegexp = /\S/;
 
+  const DEFAULT_FONT_SIZE = 30;
+  const DEFAULT_FONT_ASCENT = 0.8;
+
```

Comment:
Can you please move these lines up, such that `NonWhitespaceRegexp` and `isAllWhitespace` are still "together" here?